server/authentication/govukOneLogin.ts

```diff
@@ -50,7 +50,7 @@ async function init(): Promise<Client> {
   )
 
   const verify: StrategyVerifyCallbackUserInfo<UserinfoResponse> = (tokenSet, userInfo, done) => {
-    logger.info(`GOV.UK One Login user verified: sub =  ${userInfo.sub}`)
+    logger.info(`GOV.UK One Login user verified, sub: ${userInfo.sub}`)
```

Comment:
Not sure if we want to use `logger.debug()` as I think that only prints it out when on dev 🤔 Probably wrong though.

Reply:
I'll have a look at what `logger.debug()` does - haven't experimented with it before. Will sort out what we log (if anything ) about users once we actually start doing something with the returned data.